packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/launcher/DevLauncherReactHost.kt

```diff
@@ -81,6 +82,7 @@ object DevLauncherReactHost {
         object : ModulesProvider {
           override fun getModulesList() =
             listOf(
+              DevLauncherInternalModule::class.java,
```

Comment:
ExpoModules core is not initializing any Expo Modules in the dev launcher app, so we need to manually list them here

Reply:
interesting! didn't aware that DevLauncherInternalModule is for dev launcher app. now i'm wondering how it setup on ios

Reply:
ok unlike android, on ios we actually registers all native modules from app also for dev launcher app.